projects/js-packages/shared-extension-utils/package.json

```diff
@@ -31,6 +31,7 @@
 		"lodash": "4.17.21"
 	},
 	"devDependencies": {
+		"@automattic/jetpack-js-tools": "workspace:*",
```

Comment:
Why is this even here? Everything else seems ok with picking it up from the monorepo root.